applications/wp_k8s.md

```diff
@@ -0,0 +1,239 @@
+---
+layout: page
+title: "Deploying WordPress and MySQL with Portworx Persistent Volumes by Kubernetes"
+keywords: portworx, Wordpress, application stack, kubernetes, Persistent Storage, Persistent Volume, Persistent Volume Claim. 
+sidebar: home_sidebar
+redirect_from:
+meta-description: "Wordpress solution with Kubernetes. Use PX volume driver to create new volumes or reuse existing ones."
+---
+
+Deploying WordPress and MySQL with Portworx Persistent Volumes by Kubernetes
+This documentation explains about how to deploy a WordPress site and a MySQL database using kubernetes. 
+
+A PersistentVolume (PV) is a piece of storage in the cluster that has been provisioned by an administrator, and a PersistentVolumeClaim (PVC) is a set amount of storage in a PV. PersistentVolumes and PersistentVolumeClaims are independent from Pod lifecycles and preserve data through restarting, rescheduling, and even deleting Pods in kubernetes.
+
+Note: The files provided in this tutorial are using beta Deployment APIs and are specific to kubernetes version 1.8 and above. If you wish to use this tutorial with an earlier version of Kubernetes, please update the beta API appropriately, or reference earlier versions of kubernetes tutorial.
+
+## A.	 Create Porworx PersistentVolume
+Kubernetes supports many different types of PersistentVolumes, this step covers portworx volume. Both applications WordPress and MySQL uses portworx as PersistentVolumes and PersistentVolumeClaims to store data.
+
+## 1.	Create MySQL Portworx PersistentVolume(PV) and PersistanctVolumeClaim(PVC)- mysql-vol.yaml
+```
+apiVersion: storage.k8s.io/v1beta1
+kind: StorageClass
+metadata:
+  name: portworx-sc-repl3
+provisioner: kubernetes.io/portworx-volume
+parameters:
+  repl: "3"
+  priority_io: "high"
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: mysql-pvc-1
+  annotations:
+    volume.beta.kubernetes.io/storage-class: portworx-sc-repl3
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 2Gi
+```
+
+
+## 2.	Create WordPress Portworx PersistentVolume(PV) and PersistanctVolumeClaim(PVC)- wordpress-vol.yaml
+```
+apiVersion: storage.k8s.io/v1beta1
+kind: StorageClass
+metadata:
+  name: portworx-sc-repl3-shared
+provisioner: kubernetes.io/portworx-volume
+parameters:
+  repl: "3"
+  priority_io: "high"
+  shared: "true"
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: wp-pv-claim
+  labels:
+    app: wordpress
+  annotations:
+    volume.beta.kubernetes.io/storage-class: portworx-sc-repl3-shared
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 1Gi
+```
+
+## B.	 Create a Secret for MySQL Password
```

Comment:
I just looked at the formatting and it's confusing. See screenshot below. Heading B is for secrets but the numbering continues for the sub-section after that. Also they need to be at different heading levels.

![screen shot 2018-01-17 at 1 36 20 pm](https://user-images.githubusercontent.com/26801153/35068372-b58852c2-fb8b-11e7-903a-94e0f236a510.png)

Reply:
Please let me know, If there anything need include or update it.